http-clients/src/main/java/com/palantir/remoting/http/FeignClientFactory.java

```diff
@@ -124,6 +128,8 @@ public static FeignClientFactory of(
                 .client(client)
                 .retryer(target)
                 .options(options)
+                .logger(new Slf4jLogger(FeignClientFactory.class))
```

Comment:
I'd prefer we use a different string than FeignClientFactory, and should probably encode it as a constant.